Makefile.common.mk

```diff
@@ -22,19 +22,19 @@
 IMG = docker.io/sdake/build-tools:2019-08-03
 UID = $(shell id -u)
 PWD = $(shell pwd)
-GOBIN ?= $(GOPATH)/bin
+GOBIN_SOURCE ?= $(GOPATH)/bin
+GOBIN ?= /work/out/bin
```

Comment:
I think it's fine to get thing running again but could you leave a TODO or create an issue to fix this properly?

Reply:
Sounds fair.  I created an issue instead of TODO so this can be  tracked properly.

https://github.com/istio/istio/issues/16056